prow/config.yaml

```diff
@@ -856,6 +856,59 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0
 
+  istio-releases/daily-release:
+
+  - name: daily-release-qualification
+    agent: kubernetes
+    context: prow/qualify.sh
```

Comment:
currently the name of job needs to match the prow script

Reply:
Got it